app_usb_aud_xk_evk_xu316_extrai2s/CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+cmake_minimum_required(VERSION 3.21)
+include($ENV{XMOS_CMAKE_PATH}/xcommon.cmake)
+project(app_usb_aud_xk_evk_xu316_extrai2s)
+
+set(APP_HW_TARGET XK-EVK-XU316)
+include(${CMAKE_CURRENT_LIST_DIR}/../deps.cmake)
+list(APPEND APP_DEPENDENT_MODULES "lib_xud"
```

Comment:
lib_xud should not be required here (lib_xud is a dependency of lib_xua)

Reply:
Fixed - both in CMakeLists.txt and Makefile.